taxonomies/categories.txt

```diff
@@ -78375,6 +78375,9 @@ ciqual_food_code:en:11061
 ciqual_food_name:en:Poppy, seed
 ciqual_food_name:fr:Pavot, graine
 
+en:Product with added sugars
```

Comment:
I don't think it should be a category. Otherwise we would need categories for pretty much "product with [some ingredient]".